src/frontend/Komponenter/Journalføring/JournalføringKlageApp.tsx

```diff
@@ -0,0 +1,237 @@
+} from './journalføringUtil';
+import JournalføringWrapper, {
+    FlexKnapper,
+    Høyrekolonne,
+    JournalføringAppProps,
+    Kolonner,
+    SideLayout,
+    Venstrekolonne,
+} from './JournalføringWrapper';
+import JournalføringPdfVisning from './JournalføringPdfVisning';
+import {
+    JournalføringKlageStateRequest,
+    useJournalføringKlageState,
+} from '../../App/hooks/useJournalføringKlageState';
+import { useHentKlagebehandlinger } from '../../App/hooks/useHentKlagebehandlinger';
+import BehandlingKlageInnold from './BehandlingKlageInnold';
+import { Klagebehandlinger } from '../../App/typer/klage';
+import { Fagsak } from '../../App/typer/fagsak';
+import { useQueryParams } from '../../App/hooks/felles/useQueryParams';
+import { FamilieDatovelger } from '@navikt/familie-form-elements';
+import { useToggles } from '../../App/context/TogglesContext';
+import { ToggleName } from '../../App/context/toggles';
+import { harValgtNyKlageBehandling } from './journalførBehandlingUtil';
+import { erGyldigDato } from '../../App/utils/dato';
+import styled from 'styled-components';
+
+const KlageMottatt = styled.div`
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+`;
+
+const validerJournalføringState = (
+    journalResponse: IJojurnalpostResponse,
+    journalpostState: JournalføringKlageStateRequest
+): string | undefined => {
+    if (!journalpostState.behandling) {
+        return 'Du må velge en behandling for å journalføre';
+    } else if (!harTittelForAlleDokumenter(journalResponse, journalpostState.dokumentTitler)) {
+        return 'Mangler tittel på et eller flere dokumenter';
+    } else if (journalResponse.journalpost.tema !== 'ENF') {
+        return 'Tema på journalføringsoppgaven må endres til «Enslig forsørger» i Gosys før du kan journalføre dokumentet i EF Sak';
+    } else if (
+        harValgtNyKlageBehandling(journalpostState.behandling) &&
+        !journalResponse.journalpost.datoMottatt &&
+        (!journalpostState.behandling.mottattDato ||
+            !erGyldigDato(journalpostState.behandling.mottattDato))
+    ) {
+        return 'Mangler gyldig mottatt dato';
+    } else {
+        return undefined;
+    }
+};
+
+export const JournalføringKlageApp: React.FC = () => {
+    return <JournalføringWrapper komponent={JournalføringAppContent} />;
+};
+
+const utledValgtFagsak = (fagsak: Ressurs<Fagsak>): keyof Klagebehandlinger | undefined => {
+    if (fagsak.status !== RessursStatus.SUKSESS) {
+        return undefined;
+    }
+    switch (fagsak.data.stønadstype) {
+        case Stønadstype.OVERGANGSSTØNAD:
+            return 'overgangsstønad';
+        case Stønadstype.BARNETILSYN:
+            return 'barnetilsyn';
+        case Stønadstype.SKOLEPENGER:
+            return 'skolepenger';
+    }
+};
+
+const JournalføringAppContent: React.FC<JournalføringAppProps> = ({
+    oppgaveId,
+    journalResponse,
+}) => {
+    const { toggles } = useToggles();
+    const query: URLSearchParams = useQueryParams();
```

Comment:
`useParams<IEtellerAnnet>().noko as string;`

useParams finnes i react router :)

Reply:
Den gjelder kun for path params